Nodejs/MigrateToJsps/Jsps/EsprojFileWriter.cs

```diff
@@ -110,9 +114,14 @@ private static JObject ReadPackageJson(string projectDir)
 
         private static string GetSdkVersion()
         {
-            // TODO: figure out if sdk is installed on machine?
-            // do i need to do that or will VS automatically install given version
-            throw new NotImplementedException();
+            // Use the installed version of JSPS on the NuGet fallback folder.
+            var versions = Directory.GetDirectories(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86), "Microsoft Visual Studio", "Shared", "NuGetPackages", "microsoft.visualstudio.javascript.sdk"));
+            var newestVersion = versions
+                .Select(v => new { Version = new Version(Path.GetFileName(v)) })
```

Comment:
This should have some error handling (`Version.TryParse`). Nuget packages can have suffixes (our JSPS package used to have the `-alpha` suffix) which I think would throw an exception here.

Reply:
Yeah it did! Fixed it now. I don't think we're using that anymore, right?